pkg/install/install.go

```diff
@@ -266,9 +269,12 @@ func (i *Installer) runSteps(ctx context.Context, steps []interface{}) error {
 			if err != nil {
 				i.gatherFailureLogs(ctx)
 				if _, ok := err.(*api.CloudError); !ok {
-					err = fmt.Errorf("%s: %s", runtime.FuncForPC(reflect.ValueOf(step).Pointer()).Name(), err)
```

Comment:
this looked to me like a typo, the correct value to build the error is `step.f`

Reply:
What is the result you seeing and what is the new value?

Reply:
Right after `case condition:` line there is an existing log line (not part of my changes) referencing `step.f` but the error was referencing `step` in here, that's why I think it's a typo

Reply:
this is a bug, i have run into it. Please make sure it gets in

Reply:
I see value for this one, but all this additional logging looks overkill. I agree we need errors (https://github.com/Azure/ARO-RP/issues/790) but If it runs succ, does this information gives you anything? We don't log each successful function in the code.

an example output would be good to see how this verbose looks like. I assume you ran it and test it? causing failure and success?